aptos-move/framework/aptos-experimental/sources/trading/market/market.move

```diff
@@ -81,9 +82,15 @@ module aptos_experimental::market {
         market: address,
         // TODO: remove sequential order id generation
         last_order_id: u64,
+        config: MarketConfig,
```

Comment:
we should change Market and MarketConfig to enum

Reply:
Yes, let's do a sweep across the codebase where needed.